config/izum_vega.py

```diff
@@ -0,0 +1,119 @@
+from os import environ
+username = environ.get('USER')
+
+# This is an example configuration file
+site_configuration = {
+    'general': [
+        {
+            'remote_detect': True,
+        }
+    ],
+    'systems': [
+        {
+            'name': 'vega',
+            'descr': 'Vega, a EuroHPC JU system',
+            'modules_system': 'lmod',
+            'hostnames': ['vglogin*','cn*','gn*'],
+            'stagedir': f'reframe_runs/staging',
+            'outputdir': f'reframe_runs/output',
+            'partitions': [
+                {
+                    'name': 'cpu',
+                    'scheduler': 'slurm',
+                    'prepare_cmds': [
+                        'source /cvmfs/pilot.eessi-hpc.org/latest/init/bash',
+                        'export SLURM_EXPORT_ENV=ALL',
+                        # Avoid https://github.com/EESSI/software-layer/issues/136
+                        # Can be taken out once we don't care about old OpenMPI versions anymore (pre-4.1.1)
+                        'export OMPI_MCA_pml=ucx',
+                    ],
+                    'launcher': 'mpirun',
```

Comment:
I tried this, and then got GROMACS tests failing with 
```
srun: error: timeout waiting for task launch, started 0 of 256 tasks
```
Possibly [this](https://bugs.schedmd.com/show_bug.cgi?id=4115) issue.

I switched back to `mpirun`, but I _did_ add `SLURM_MPI_TYPE=pmix`. This means that to do CPU autodetection, we _only_ need to switch `mpirun` to `srun` again.

Note however that I now get errors with `mpirun` for multinode stuff as well.

```
An ORTE daemon has unexpectedly failed after launch and before
communicating back to mpirun. This could be caused by a number
of factors, including an inability to create a connection back
to mpirun due to a lack of common network interfaces and/or no
route found between them. Please check network connectivity
(including firewalls and network routing requirements).
```
I'm thinking they may have messed something up on the system, they did have an issue last week with the filesystem. Maybe they also unintentionally made other changes... I'll dive further into it if I have time, but I propose we don't let this `srun` vs `mpirun` choice block this PR.